mysql/infoschema_test.go

```diff
@@ -57,6 +57,10 @@ func TestProcessInfoSchemaMYSQL(t *testing.T) {
 			rows: [][]driver.Value{
 				{"productid", "PRIMARY KEY"},
 				{"userid", "PRIMARY KEY"}},
+		}, {
+			query: "SELECT (.+) FROM INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS (.+)",
+			args:  []driver.Value{"test", "cart"},
+			cols:  []string{"REFERENCED_TABLE_NAME"},
```

Comment:
Shouldn't we add a test that actually exercises the foreign key code?
This change to the test (and the following two changes) will just return empty rows for getRefTables and basically we'll never call getForeignKey. So we'll never really test the foreign key code.

Reply:
The only test for foreignKey would be to check `conv.ForeignKeys` after adding some test cases. If you take a look at PR note, I have mentioned it as pending and it will be done.

Apologies if it was not clear.

Reply:
tests for info schema and dump are added

Reply:
Thanks Pradip!
Sorry I missed the PR note -- I'd assumed the PR was essentially complete and ready for final review.